Common/linked_list.h

```diff
@@ -0,0 +1,44 @@
+#ifndef PAS_LINKED_LIST_H
+#define PAS_LINKED_LIST_H
+
+#include "common.h"
+#define bool char
+
```

Comment:
```suggestion
#define true 1
#define false 0
```

Reply:
You have to remove this 2 line from Common.h

Reply:
flemme